soft4pes/control/mpc/solvers/mpc_bnb.py

```diff
@@ -7,18 +7,18 @@
 
 class MpcBnB:
     """
-    Branch-and-bound solver for Model predictive control (MPC).
+    Branch-and-bound (BnB) solver for model predictive control (MPC).
 
     Attributes
     ----------
     J_min : float
         Minimum cost.
-    u_seq : 1 x 3*Np ndarray
-        Array for the 3-phase switch position sequence with the lowest cost.
-    u_temp : 1 x 3*Np ndarray
-        Temporary array for 3-phase switch position sequence.
+    U_seq : 1 x 3*Np ndarray
+        Array for sequences of three-phase switch positions (switching sequences) with the lowest cost.
```

Comment:
I have to ask, you modified this to be sequences. Isn't it that we have one sequence of three-phase switch positions? In the enumeration, we store multiple sequences. Is this a mistake or do I understand something wrong?

Reply:
u(k) is a three-phase switch position
if we have {u(k), u(k+1), u(k+2), ...} we have a switching sequence